### PR TITLE
core: remove disableDeviceEmulation flag

### PR DESCRIPTION
**Pull request by wardpeet**
**Summary**
Remove disableDeviceEmulation flag and fore people on emulatedFormFactor

**Related Issues/PRs**
closes https://github.com/GoogleChrome/lighthouse/issues/7044

### CODE REVIEW

**Comment by wardpeet** [comment]
🤦‍♂️ sorry, I fixed it. I was so focused on the javascript flag and not considering the cli flag.

**Comment by wardpeet** [comment]
I could add the deprecation and mvoe this one to version 6? As I don't think it's worth it. It's a major bump but it's always nice to tell people what's going away.

**Comment by patrickhulce** [comment]
@wardpeet oh yeah I was thinking if anything we cherry-pick a deprecation notice onto 4.3 for a final patch release or something, agreed not worth bumping to v6 👍

**Comment by wardpeet** [comment]
oh yeah let me create a small PR for it.

File: lighthouse-cli/cli-flags.js [hunk 1]
```diff
@@ -123,7 +122,7 @@ function getFlags(manualArgv) {
 
       // boolean values
       .boolean([
-        'disable-storage-reset', 'disable-device-emulation', 'save-assets', 'list-all-audits',
```
Comment (brendankenny) [inline lighthouse-cli/cli-flags.js]: nbd, just the flag :)

**Pull request merged**